### PR TITLE
Bug fixes: 1) Actually use stringer.lta method 2) prevent crash without book values 3) minor bug fixes related to plots and qml

### PR DESCRIPTION
**Pull request by koenderks**
Relates to https://github.com/jasp-stats/jasp-issues/issues/1588, but also contains more bug fixes for 0.16.1.
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1738
- Closes https://github.com/jasp-stats/jasp-test-release/issues/1740

### CODE REVIEW

File: tests/testthat/_snaps/auditbayesianplanning/prior-and-expected-posterior-distribution-1.svg [hunk 1]
```diff
@@ -29,8 +29,6 @@
...
-<circle cx='124.15' cy='481.54' r='5.12' style='stroke-width: 1.70; fill: #BEBEBE;' />
```
Comment (vandenman) [inline tests/testthat/_snaps/auditbayesianplanning/prior-and-expected-posterior-distribution-1.svg:32]: These two points are now missing in all the prior-and-expected-posterior plots. Is that intended?

**Review by vandenman — CHANGES_REQUESTED** [review]
Changes look fine, just a small question thing about the missing points.

**Comment by koenderks** [comment]
@vandenman yes this is intended. These points should now only show in the prior and (not expected) posterior distribution plot

**Review by vandenman — APPROVED** [review]
both issues are fixed!

**Pull request merged**